components/Image/style.scss

```diff
@@ -72,11 +72,7 @@
   }
 
   figure .img {
-    display: inline-block;
-
-    > img:only-of-type {
-      display: block;
-    }
+    display: block;
   }
```

Comment:
Hard to test here, but I think this might break some other common image use cases, such as icon alignments. I'm hesitant to merge this, though I don't know another way to spin up an integration test. @kellyjosephprice—have you tested this locally inside of ReadMe-proper?

Reply:
Yea, I tested with it linked into the UI library. The following markdown looks okay to me (except that the UI library isn't serving emojis write now):
```
before

[block:image]
{
  "images": [
    {
      "image": [
        "http://placekitten.com/400/200",
        null,
        "WTF Kitty"
      ],
      "align": "center",
      "caption": "WTF Kitty"
    }
  ]
}
[/block]

after

Foobar ![](http://placekitten.com/200/200)

asdf :-1:
```

![image](https://github.com/readmeio/markdown/assets/451488/6d1cb6e0-a71c-4b7d-b79b-4a1fddcbd362)


Got any other cases to test?

---

And it looks like we only wrap it in a `figure` if it has a caption? https://github.com/readmeio/markdown/blob/next/processor/parse/magic-block-parser.js#L120-L133

Reply:
> *it looks like we only wrap it in a figure if it has a caption?*

ahhhhh, I believe that is correct. good call on that.

Reply:
Probably a rare case, but how about a `[align=right]` image *with* a caption?

Reply:
Ugh, only saw this after merging. Definitely not good.

Reply:
I actually think it might be fine? Depends on how the float is applied (i.e. to the `img` itself, or to the `.img` parent?)